src/utils/ValidationUtils.ts

```diff
@@ -39,5 +60,100 @@ export const useValidationUtils = () => {
         }
     ];
 
-    return { requiredFieldRules, requiredSelectionRules, doiValidationRules, uriValidationRules };
+    const isbnPattern = /^(?:(?:\d[ |-]?){9}[\dX]|(?:\d[ |-]?){13})$/i;
+    const isbnValidationRules = [
+        (value: string) => {
+            if (!value || value.trim() === "") return true;
+            if (isbnPattern.test(value)) return true;
+            return invalidISBNMessage.value;
+        }
+    ];
+
+    const eIssnPattern = /^\d{4}-\d{4}$/i;
+    const eIssnValidationRules = [
+        (value: string) => {
+            if (!value || value.trim() === "") return true;
+            if (eIssnPattern.test(value)) return true;
+            return invalidEISSNMessage.value;
+        }
+    ];
+
+    const printIssnPattern = /^\d{4}-\d{4}([\dX])?$/i;
+    const printIssnValidationRules = [
+        (value: string) => {
+            if (!value || value.trim() === "") return true;
+            if (printIssnPattern.test(value)) return true;
+            return invalidPrintISSNMessage.value;
+        }
+    ];
+
+    const scopusAfidPattern = /^\d+$/i;
+    const scopusAfidValidationRules = [
+        (value: string) => {
+            if (!value || value.trim() === "") return true;
+            if (scopusAfidPattern.test(value)) return true;
+            return invalidScopusAfidMessage.value;
+        }
+    ];
+
+    const apvntPattern = /^\d+$/i;
+    const apvntValidationRules = [
+        (value: string) => {
+            if (!value || value.trim() === "") return true;
+            if (apvntPattern.test(value)) return true;
+            return invalidApvntMessage.value;
+        }
+    ];
+
+    const eCrisIdPattern = /^\d+$/i;
+    const eCrisIdValidationRules = [
+        (value: string) => {
+            if (!value || value.trim() === "") return true;
+            if (eCrisIdPattern.test(value)) return true;
+            return invalidECrisIdMessage.value;
+        }
+    ];
+
+    const eNaukaIdPattern = /^[a-zA-Z]{2}\d+$/i;
```

Comment:
I believe it will be always capitalized letters at the beginning, can you try to find a couple of examples at enauka web site. The issue might be that existIdCheck at the backend is case sensitive, meaning AV123 and av123 are two different IDs.

Reply:
Fixed.